example/harris.cpp

```diff
@@ -0,0 +1,235 @@
+#include <boost/gil/image.hpp>
+#include <boost/gil/image_view.hpp>
+#include <boost/gil/extension/io/png.hpp>
+#include <boost/gil/image_processing/numeric.hpp>
+#include <boost/gil/image_processing/harris.hpp>
+#include <vector>
+#include <functional>
+#include <set>
+#include <iostream>
+#include <fstream>
+
+namespace gil = boost::gil;
+
+gil::gray8_image_t to_grayscale(gil::rgb8_view_t original) {
```

Comment:
Please add a comment why the default color conversion is not suitable here.
Otherwise it raises obvious question.
(Less questions raised, higher code quality.)

Reply:
(formatting nit-pick: and please put curly brackets onto their own lines.)